staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -6263,8 +6341,12 @@ func TestValidateCustomResourceDefinitionValidation(t *testing.T) {
 					Type: "null",
 				},
 			},
-			opts:      validationOptions{requireValidPropertyType: true, requireStructuralSchema: true},
-			wantError: true,
+			opts: validationOptions{requireValidPropertyType: true, requireStructuralSchema: true},
+			expectedErrors: []validationMatch{
+				unsupported("spec.validation.openAPIV3Schema.type"),
+				forbidden("spec.validation.openAPIV3Schema.type"),
+				invalid("spec.validation.openAPIV3Schema.type"),
+			},
```

Comment:
we really get all three errors?

Reply:
Yes. We get:
- `Invalid value: "null": must be object at the root` from checking the root type;
- `Forbidden: type cannot be set to null, use nullable as an alternative` from [nullable check](https://github.com/kubernetes/kubernetes/blob/763916aa2d7cca2f334037785e4d84136fdf6d11/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L1040)
- `Unsupported value: "null": supported values: "array", "boolean", "integer", "number", "object", "string"` for type supported value check

Reply:
Put this info in the test code as comments as well as which one is the CEL error (since some are from other parts of the CRD validation system)?